AssetRipper.Fundamentals/Configuration/CoreConfiguration.cs

```diff
@@ -86,7 +121,7 @@ public void SetProjectSettings(UnityVersion version, BuildTarget platform, Trans
 		public virtual void ResetToDefaultValues()
 		{
 			ScriptContentLevel = ScriptContentLevel.Level2;
-			StreamingAssetsMode = StreamingAssetsMode.Extract;
+			IgnoreStreamingAssets = false;
```

Comment:
Why did you change this?

Reply:
IgnoreStreamingAssets resolves the enum value in its getter so that we can use GetSettings and SetSettings to store its value (https://github.com/AssetRipper/AssetRipper/pull/513/files/b3ceb5272787dd1b17ca43ce1c68b7690a1ce10b#diff-7581cca434414659a49634ea9b50525e26a8a415750b6e8b7b7e1a0e23e9ad0aR55)